From 29579082da3012d55167546739751eedb72b29bd Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Sat, 16 May 2020 18:20:57 +0300
Subject: [PATCH] [SW-12964] goya2: pldm: update uart frequency

Use pll div3 frequency set by btl in ppboot uart also.
frequency :250 Mhz, actual on pldm: 20Khz

Change-Id: Ibd4194ffdabb44892cdb3d3d72b15fd9f7f5a903
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/goya2/pldm.c b/goya2/pldm.c
index 4408e08..fe13594 100644
--- a/goya2/pldm.c
+++ b/goya2/pldm.c
@@ -12,7 +12,7 @@
 #include <uart.h>
 
 #define SYS_CLOCK_FREQUENCY	155000ULL
-#define UART_CLOCK_FREQUENCY	(SYS_CLOCK_FREQUENCY / 5)
+#define UART_CLOCK_FREQUENCY	20000ULL
 #define PLDM_UART_BAUDRATE	1200
 
 u32 hl_skip_pcie_init(void)
